src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -56,8 +56,9 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
             await spectralManager.InstallSpectralAsync(cancellationToken);
             await oasdiffManager.InstallOasdiffAsync(cancellationToken);
 
-            var swaggerDocPaths = await swaggerManager.RunSwaggerAsync(this.OpenApiSwaggerDocumentNames, cancellationToken);
-            await spectralManager.RunSpectralAsync(swaggerDocPaths, this.OpenApiSpectralRulesetUrl, cancellationToken);
+            var generateOpenApiDocsPath = (await swaggerManager.RunSwaggerAsync(this.OpenApiSwaggerDocumentNames, cancellationToken)).ToList();
+            await spectralManager.RunSpectralAsync(generateOpenApiDocsPath, this.OpenApiSpectralRulesetUrl, cancellationToken);
```

Comment:
I believe that Spectral and Oasdiff could run simultaneously? Behavior on error should be the same than before

```csharp
var runSpectralTask = spectralManager.RunSpectralAsync(generateOpenApiDocsPath, this.OpenApiSpectralRulesetUrl, cancellationToken);
var runOasdiffTask = oasdiffManager.RunOasdiffAsync(this.OpenApiSpecificationFiles, generateOpenApiDocsPath, cancellationToken);

await runSpectralTask;
await runOasdiffTask;
```